control-plane/connect-inject/endpoints_controller.go

```diff
@@ -635,11 +731,49 @@ func (r *EndpointsController) createServiceRegistrations(pod corev1.Pod, service
 				}
 			}
 		}
-	}
 
+	}
 	return service, proxyService, nil
 }
 
+// redirectTrafficAnnotation creates an annotation that contains iptables configuration.
+func (r *EndpointsController) redirectTrafficAnnotation(pod corev1.Pod, cfg iptables.Config) error {
```

Comment:
I think `addTrafficRedirectAnnotation` is clearer.

Reply:
Done.